content/blog/2018/12/2018-12-10-the-official-Docker-image.adoc

```diff
@@ -0,0 +1,30 @@
+---
+:layout: post
+:title: Official Jenkins image to use from Docker Hub
+:tags:
+- docker
+:author: batmat
+---
+
+There a now three different Docker Hub repositories that are or have been used as the "official" Jenkins image.
+This article aims at providing a clarification about which one is the current official one (as of December 2018 :-)).
+
+## The official one
+
+[source]
+docker pull jenkins/jenkins
```

Comment:
Showing the right repo link could be better.
https://hub.docker.com/r/jenkins/jenkins/

Reply:
Mixed feeling. At the same time, nobody uses the full url apart from web browsing. I wanted at least one occurrence of the right docker pull/run command with the right image name.

But agreed, should find a way to put this URL somewhere.